### DISCUSSION TITLE
Latest Beta release flagged as `Trojan`

### DISCUSSION THREAD

**Discussion by vedxp** [LawnchairLauncher/lawnchair#4180] [Q&A]
The latest released beta, `14.0.0.Beta.1` is being flagged as `Trojan` on VirusTotal.

I wonder this is just a **false-positive**, but still need to confirm if there's any chance it might get fixed.

Here's the link to the scan result: https://www.virustotal.com/gui/file/9e529287ea06cb254a79660566db5eda4cc4500d73fb73b1ee9bd369bebf3894/summary

**Comment by vedxp** [comment]
Edit: It's the same with the releases compiled by @Goooler also.
Here's the scan result of his latest build (14.0.0-20240326):
https://www.virustotal.com/gui/file/58f35d5f707938ec4a4a4556daaa301b869f1adeae35992da764e9c691141d5c

**Comment by RustoMCSpit** [comment]
K7GW
Trojan ( 004a8ed81 )

**Comment by RustoMCSpit** [comment]
https://github.com/LawnchairLauncher/lawnchair/discussions/3899 duplicate

**Comment by validcube** [comment]
VirusTotal isn't meant to replace your traditional antivirus, it's simply analyse and aggregate result from security providers which have different detection tactics and verdict for user to consume. My opinionated view is to look for any detection more than 5~8 from (usually enterprise also works too but they're likely too aggressive with their detectors) reputable antiviruses if more then it should be counted as suspicious. (That could be what you intended to do, but you'll never know, maybe you just wanted to intentionally host some files but the antivirus just flagged it because you're not supposed to do it.)
